### ISSUE TITLE
[Feature Request] Option to only show installed games during search

### ISSUE THREAD

**Issue by darklinkpower** [felixkmh/QuickSearch-for-Playnite#2]
I have found that it can be difficult to find some games to play because it mixes games that are uninstalled in the list so this should help. Personally I think only installed games should be in the search by default because you are not installing games constantly but not sure.

For example, typing `Dark Souls` gives me an uninstalled and unplayable game

![image](https://user-images.githubusercontent.com/1389286/119721718-1599ad80-be31-11eb-99d9-058c3a934714.png)

**Comment by felixkmh** [comment]
I mean its kind of the point of this to find everything there is, so I would not want to filter out not installed games completely. I'm thinking I could add an option to reorder game results such that it pushes one or more (max number set by the user) installed games in front of the first not installed game. This would always give you a playable game as the first result if available, but still keep highly matching uninstalled games easily accessible.

**Comment by darklinkpower** [comment]
Yeah anything that helps see installed games first is appreciated.

**Comment by felixkmh** [comment]
For the time being I added a new option to sort by installation status before the name if scores are equal in version 1.2.
